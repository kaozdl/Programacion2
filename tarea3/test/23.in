# filtrar_binario
filtrar_binario 12 > .
insertar_binario (15,q) .
filtrar_binario 12 > .
filtrar_binario 15 = .
filtrar_binario 22 < .
filtrar_binario 22 > .
insertar_binario (14,c) .
insertar_binario (16,d) (13,t) (-3,c2) (9,c1) .
filtrar_binario 15 < .
filtrar_binario 12 > .
filtrar_binario 22 > .
Fin
