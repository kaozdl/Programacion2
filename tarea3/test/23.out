1>
2>
3>
(15,q)
0
0
4>
(15,q)
5>
(15,q)
6>
(15,q)
7>
8>
(15,q)
-(14,c)
1
0
9>
-(13,t)
(15,q)
--(16,d)
---(-3,c2)
----(9,c1)
-(14,c)
4
1
10>
-(13,t)
(-3,c2)
--(9,c1)
-(14,c)
11>
-(13,t)
(15,q)
--(16,d)
-(14,c)
12>
13>Fin.
